### DISCUSSION TITLE
Taiga

### DISCUSSION THREAD

**Discussion by mahes1287** [coollabsio/coolify#2585] [Service Template Requests]
Can u integrate taiga service as one click installation

https://taiga.io/

**Comment by MajidRaimi** [comment]
This would be helpful tbh <3

**Comment by MajidRaimi** [comment]
Did you found anyway to install it?

**Comment by mahes1287** [comment]
@MajidRaimi i couldnt found the way to install with coolify. due to time constraint i went with docker installation as per taiga guideline.

**Reply by saurabh-vijayvargiya** [reply]
@mahes1287 I also tried to do it with docker but it kept failing. Can you share the steps that you followed while installing it on the coolify?

Thanks :)

**Reply by mahes1287** [reply]
I followed taiga community guidelines...

**Comment by fmendezy** [comment]
Please add Taiga.... alternative to trello....

**Comment by artofactbe** [comment]
Yes please!

**Comment by svenbledt** [comment]
Bump +1
